docker-compose.yml

```diff
@@ -40,7 +40,8 @@ services:
     volumes:
 # Database migration should be done manually, so .tmp -> .db:
       - './.db/pg:/var/lib/postgresql/data'
-
+    ports:
```

Comment:
Are you sure this ports definition is needed?
(For me everything is fine without it also.)